src/main/java/com/loadero/model/LoaderoCollection.java

```diff
@@ -0,0 +1,26 @@
+package com.loadero.model;
+
+import java.util.List;
+
+public abstract class LoaderoCollection<T> {
```

Comment:
do we really need to define this? with simple "readAll" this looks like just a duplicate of `List<T>`. we could later implement paginated results (that read only part of the resources, not all of them), but until then it kind of seems unnecessary to have this class